src/OpenLoco/Audio/Audio.cpp

```diff
@@ -1132,20 +1132,12 @@ namespace OpenLoco::Audio
     // previously called void playTitleScreenMusic()
     bool playMusic(PathId sample, int32_t volume, bool loop)
     {
-        static PathId currentTrackPathId;
-
         auto* channel = getChannel(ChannelId::music);
         if (!_audioInitialised || _audioIsPaused || !_audioIsEnabled || channel == nullptr)
         {
             return false;
         }
 
-        if (currentTrackPathId == sample)
```

Comment:
this was the core problem - going into the scenario editor didn't clear this variable, so when re-entering the title screen, the value hadn't changed, and thus this method returned without ever playing the music